core/src/main/scala/kafka/tools/ConsoleProducer.scala

```diff
@@ -237,6 +241,10 @@ object ConsoleProducer {
             .withRequiredArg
             .describedAs("producer_prop")
             .ofType(classOf[String])
+    val producerConfigOpt = parser.accepts("producer.config", "Producer config properties file.")
```

Comment:
I think the inconsistency with other options is better than inconsistency with every other tool. I suggest we look at making an improvement across all tools after 0.9.0.0.